src/transformers/models/bort/modeling_bort.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
```

Comment:
(nit) we could add some examples here as well similar to how it's done for MT5: https://github.com/huggingface/transformers/blob/c19d04623eacfbc2c452397a5eda0fde42db3fc5/src/transformers/models/mt5/modeling_mt5.py#L33

I think for `BortModel` we can just show how to get the `last_hidden_state` and for all other models we could show how to get the loss for fine-tuning.